README.md

```diff
@@ -71,6 +72,16 @@ $ npm install @sap-ai-sdk/langchain
 
 We have created a sample project demonstrating the different clients' usage of the SAP Cloud SDK for AI for TypeScript/JavaScript. The [project README](./sample-code/README.md) outlines the set-up needed to build and run it locally.
 
+## Local Testing
+
+To test SAP Cloud SDK for AI features locally while building your application, follow these steps:
+
+- Download a service key for your AI Core service instance.
+- Create a `.env` file in the root of your application.
+- Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
+
+By adding the `.env` file, the SDK uses the service key to interact with the AI Core service. This setup allows you to test clients such as orchestration and OpenAI locally, provided you have deployments for orchestration and OpenAI in SAP BTP.
```

Comment:
[q/req] Is this really true? I don't think it is. Support for `.env` files usually comes through [dotenv](https://www.npmjs.com/package/dotenv) and it would need to be a dependency of our packages (which it isn't). Unless we explicitly read and parse `.env` files, which I am not aware of. Or did I miss something else?

I think users would have to make sure the environment variable is there, however they want to do this. My suggestion would be to omit the step "Create a `.env` file in the root of your application." and instead say that they should set the service key as environment variable. *How* is up to them.

Reply:
Yeah, you are right. We seem have explicitly used `dotenv` to read and populate the environment variables from the `.env` file for this work.

Reply:
Done.

Reply:
But still `By adding the `.env` file, the SDK uses the service key to interact with the AI Core service.` is not correct. SDK does not use `.env` directly. And for `sample-code` and `e2e-tests`, `.env` needs to be in the root of each package. Also we are not sure if user uses these two package for local testing (probably not using e2e).

I would instead say `The SDK parses the service key from the environment variable to interact with the AI Core service.`

Reply:
I forgot to edit this line, have made the changes now.